src/lang/en.json

```diff
@@ -102,6 +103,100 @@
   "filter_win": "Result",
   "filter_lane_role": "Lane",
   "filter_patch": "Patch",
+
```

Comment:
Preferably we'd keep the file in alphabetical order, but i guess we can fix that later